src/il_representations/test_support/configuration.py

```diff
@@ -103,7 +108,7 @@
             'algo': tune.grid_search(['bc']),
             'freeze_encoder': tune.grid_search([False])
         },
-        'env_cfg': tune.grid_search([ENV_CFG_TEST_CONFIGS[0]]),
+        'env_cfg': tune.grid_search([ENV_CFG_TEST_CONFIGS[2]]),
```

Comment:
Does it matter that this is using `ENV_TEST_CONFIGS[2]`? (i.e. will it break if we change 2 to 1 or 0?) If so, it might be a good idea to add a comment explaining why, and maybe a corresponding comment to the definition of `ENV_TEST_CONFIGS` as well.